project/Settings.scala

```diff
@@ -132,7 +132,7 @@ object Settings {
       * Specify that all tests will be executed in a single external JVM.
       * By default, tests executed in a forked JVM are executed sequentially.
       */
-    Test / fork := true,
+    Test / fork := false,
```

Comment:
Sounds good to not fork.. does this increase test execution time?

Reply:
doesn't seem like it increases execution time much..it's still around 18 min